cli/QuickstepCli.cpp

```diff
@@ -430,6 +445,13 @@ int main(int argc, char* argv[]) {
         reset_parser = true;
         break;
       }
+#ifdef QUICKSTEP_ENABLE_GOOGLE_PROFILER
+      // Profile only if profile_file_name flag is set
+      if (!started_profiling && !quickstep::FLAGS_profile_file_name.empty()) {
```

Comment:
Now we are using the singleton design pattern.

I still have a question: why we start profiling after one query finishes execution, instead of before any query starts? Thanks!